int8 joint_id   # joint index, from 0 to 11
float32 position  # Position to move to in radians
float32 time      # Finish movement time in ms
bool invert       # Servo direction invertion
